static/src/javascripts/projects/common/modules/ui/cookiesBanner.js

```diff
@@ -35,7 +35,7 @@ define([
                 euMessageCookie = cookies.get(EU_COOKIE_MSG);
             if (!euMessageCookie || euMessageCookie != 'seen') {
                 var link = 'https://www.theguardian.com/info/cookies',
-                    txt = 'Welcome to the Guardian. This site uses cookies, read our policy <a href="' + link + '" class="cookie-message__link">here</a>',
```

Comment:
Generally having a link that is understandable standalone is the most ideal so 'Read our policy here' might make more sense as the link if changing that.